pypiserver/__main__.py

```diff
@@ -117,6 +120,9 @@ def usage():
       A format-string selecting Http-Error properties to log; set to  '%s' to see them all.
       [Default: %(body)s: %(exception)s \n%(traceback)s]
 
+    --log-to-stdout
+      log messages to stdout instead of the default stderr
```

Comment:
Since we're touching this, it might be better to give people some more configurability with this, for example:

```
--log-to-stream
  Log messages to the specified stream. Valid values are "stdout", "stderr", or "none"
```

That way people can turn off string logging altogether if they want. With your refactor to the `init_logging` function, this should be pretty easy to add in. What do you think?

Reply:
Sounds good. I've added it to the PR. I've renamed it to `log-stream` to be more consistent with the `log-file` option